common/materialContainer/MaterialContainer.h

```diff
@@ -76,7 +76,7 @@ template <class T> class MaterialContainer
     , contentFreq(MC.contentFreq)
   {
   }
-  MaterialContainer& operator=(const MaterialContainer& mc) { return *this; }
+  MaterialContainer& operator=(const MaterialContainer&) { return *this; }
```

Comment:
If by "copy method" you mean an overloaded equals operator, then yeah, I don't see the reason either. No one's using this class yet, so we can reasonably remove it.